software/speller/data_collection_platform/dcp/mp/shared.py

```diff
@@ -5,5 +5,14 @@
 # creating a manager that will allow processes to be communicated to from the flask app
 # the manager will control when processes can pull from the bci device
 manager = Manager()
-# {process_id: {character, phase, frequency, q, config_id, collecting}}
+# {process_id: {
+#   character : str, 
+#   phase: float, 
+#   frequency: float, 
+#   q: Queue, 
+#   config_id: int,
+#   bci_config: str,
+# , state: str (Could be either start, ready, collect, stop)
+#   }
+# }
 bci_processes_states = manager.dict()
```

Comment:
@wykzhang I think rather than keeping it like this, we should use cookies to store a mapping from process_id to their respective shared queue and shared variables, what do you think?